locale/en_US/submission.xml

```diff
@@ -157,6 +157,10 @@
 	<message key="submission.changeComments">Change to</message>
 	<message key="submission.changeFile">Change File</message>
 	<message key="submission.citations">References</message>
+	<message key="submission.citations.description">Add a reference per line, so that they can be extracted.</message>
+	<message key="submission.parsedCitations">Parsed References</message>
+	<message key="submission.parsedCitations.description">The following references have been extracted and will be automatically linked to the submission metadata and for Crossref citation tracking (if used).</message>
```

Comment:
I'd like to see two separate messages, one for if Crossref is not active:

> The following references have been extracted and will be linked to the submission metadata.

And one if Crossref is active:

> The following references have been extracted. They will be linked to the submission metadata and included in Crossref citation tracking.

Reply:
Hmmm... I cannot for sure determine if a journal is using Crossref, so maybe to leave only that first message and then I can write in the Crossref plugin README that the extracted citations will be considered for DOI export/registration?

Reply:
What about just checking if the CrossRef plugin is enabled? Otherwise, I think your solution works well too, and will have the added benefit of isolating CrossRef concerns within the plugin itself.

Reply:
I think the import/export plugins are enabled per default. I could eventually see if the depositor name and e-mail are there, that are necessary for Crossref export/registration...

Reply:
Ah, I see. It's up to you. My main concern was to avoid showing a potentially confusing message to someone who doesn't know what Crossref is. However you think that's best address is :+1:

Reply:
Thanks a lot @NateWr! I will then leave it with that first message, without Crossref, fro now...